### DISCUSSION TITLE
Spammer Purge setup issues

### DISCUSSION THREAD

**Discussion by mrsqueakcat** [ThioJoe/YT-Spammer-Purge#802] [Q&A]
This is probably bonehead simple but I can't figure it out.  

I'm trying to set up Spammer Purge but I am having all sorts of odd issues with the google cloud platform. A lot of the pages I need to setup are either not where shown, are overlayed on top of each other making them unreadable, are halfway ghosted to where I cant see what I m looking for, or the left side sidebar is superimposed over everything so far that  I cant work with the screens I need to. 

Seems to be a repeatable problem on two separate laptops both running win 10 64 bit. 

I've tried watching the related YT videos on setup and have done my best to read every How-to I can find but nothing addresses these odd  problems.

**Comment by Firecul** [comment]
What browser are you using and have you tried it's private/incognito mode to see if it's an extension you are using?

**Reply by mrsqueakcat** [reply]
Standard google one.  

I'm trying to update my knowledge of programming and thought this would be a fun one to learn with but my present knowledge base is more geared towards industrial application tech from 10 to 20+ years ago.   I feel like I am already in over my head on something that should be simple that I already have some understanding of.

**Reply by Firecul** [reply]
I've just tried to go to a few of the pages on my pc and everything looks fine to me so I think it's something on your end.
Can you try Microsoft Edge (it should be installed on your computer already if it's up to date) just to eliminate a problem with your browser? That'd narrow down the list of things it could be.

> I feel like I am already in over my head on something that should be simple that I already have some understanding of.

Trying to learn something new whilst fighting technical issues makes things much harder, once we figure that out it'll get easier.

**Reply by mrsqueakcat** [reply]
I don't mind too much. When I worked in the oilfields I was the one tech that could magically get anything that was supposed to default unbreakable to glitch in some impossible way just by using a program the way it was supposed to be used. 

One time I accidentally remotely took over, from half a continent away, full control of a 2600 HP pump system with my work laptop sitting in our service truck on a remote well site in North Dakota, despite us having a very limited satellite link to the outside world, just by having fat-fingered a local pump system unit number we were doing diagnostics on.    Somehow my laptop could remotely access all of our greater company equipment via any WiFi to internet connection anywhere in the world for some reason.

Our upper-level company programmer techs said it was impossible until I took over their test bench simulator rig and proved it.  Their fix was to take my laptop away and give me another one.  :p

**Comment by TechStudent10** [comment]
I would recommend following ThioJoe's guide and not some random one on the internet. It's on his channel.

**Reply by mrsqueakcat** [reply]
That's what I tried.  I don't think it is a problem with his tutorials.  It seems like an interface issue with the google side of things at this point.

**Reply by TechStudent10** [reply]
I'll hopefully have time to try the guide again at some point during the weekend, but it should work.

Try it with a different account and see if it works.

**Reply by mrsqueakcat** [reply]
Thanks,  I'll give things a retry from a different angle this weekend if I get stormed in again.

**Comment by mrsqueakcat** [comment]
Here's what I have so far following thioJoes tutorial videos. 
Everything matches up to time stamp ~1:25 where he goes to create credentials in the "Welcome to the API library" screen.
My screen looks nothing like that one.  (See first attachment. )
![Screenshot (1)](https://user-images.githubusercontent.com/104110598/164990291-ef83f879-474e-477d-bd8f-cbdd562e0fb5.png)

As you can see the left side is partially cut off and I can't get that overlay to go away. 
It appears I can select the  'User Data or Public Data and the next tabs.  
There are no other tabs or anything I can scroll down to lower on the screen as his video shows. 

Hitting next gets me to this screen I cant do anything with due to everything being behind the left-hand menu bar. 
The next screen I get is this one. 
![Screenshot (2)](https://user-images.githubusercontent.com/104110598/164990297-7d03d024-f209-4781-8967-2acb45314823.png)

This is all the further I can get on setup.

**Comment by Masamune3210** [comment]
Try zooming the page out a bit, ctrl+- or ctrl+scroll wheel down should do it

**Reply by mrsqueakcat** [reply]
I tried that first.  All it does is everything just gets smaller at the same time.  
![Screenshot (3)](https://user-images.githubusercontent.com/104110598/164990342-60822e9f-776e-4302-a2ab-908efbd34e72.png)

**Comment by mrsqueakcat** [comment]
Here is what I see when I get to the Add or Remove Scopes part.
![Screenshot (4)](https://user-images.githubusercontent.com/104110598/164991818-dcc85278-db9f-45f4-92bc-4e1bb51aee49.png)

Here is the Add Users screen. 
![Screenshot (5)](https://user-images.githubusercontent.com/104110598/164992012-453908d4-2f7f-4a13-a7d2-3a4e9967c658.png)

**Comment by mrsqueakcat** [comment]
I think I found a workaround to get past the last two screen glitches but I apparently now need to purchase a cloud domain name for any of this to work. No thanks. I'm not paying for stuff if that's what's needed in the end. 

I kinda figured this whole thing was too good to be true.

**Reply by TechStudent10** [reply]
You don't need to pay. This is some kind of glitch with your browser. Try using a different one. I'm currently using Brave and it works fine.

**Reply by mrsqueakcat** [reply]
Got it working!    One problem. I ran it on a channel with several well-known trolls/bots.  Zero hits for them but several on normal people that don't do bot posting or spamming that I interact with all the time.

**Reply by mrsqueakcat** [reply]
I got it to pick up all his spam posts, now what?    I could care less about documenting all his hate-baiting spam. I just want YT to know about it and clear out the account like they are supposed to.

**Reply by mrsqueakcat** [reply]
Did some testing here.  Something ain't right.   I did some left-leaning channels with loads of well-known generic spam comment bot accounts and it barely picked up any, other than a few I know are not bots due to past positive interactions with them on other channels. 

Then I did some right-leaning channels with people I have known and interacted with in-depth for a long time and had many of them pop up as spam accounts, including myself, while all but one troll/bot account got missed.   

WTF?

**Reply by Firecul** [reply]
We can't help with out seeing more examples for this problem

**Reply by mrsqueakcat** [reply]
I was reading through the comments it flagged to see what it was picking up and noticed several user names I know are real people and it had their comments flagged that were just them, and even me, chatting with other people about whatever topic came up in whatever thread they/we were in.  Clearly not spam or negative trolling posts.  

My question now is, How to use the report function to start flagging the new bots that are popping up all over. Typing in report at the end of its cycle doesn't seem to do anything I can see.  What am I missing?

**Reply by mrsqueakcat** [reply]
Here's what I am seeing at the end of the program.  Where/how do I report things to YT or is that being done automatically at the end of the cycle and I don't have to tell it to? 
![Screenshot (6)](https://user-images.githubusercontent.com/104110598/165001804-9da0457e-7173-40ae-b1e8-d6947713e880.png)

**Reply by Firecul** [reply]
It's telling you at the bottom, that the only way you can report comments is by using the auto smart, sensitive smart and ID modes.
If you are scanning comments with an arbitrary word or phrase that can very easily be abused so is not enabled.

When you use the 3 modes I mentioned above, and have selected or narrowed down the comments you view as spam, you will then be able to type `report`. At that point, those comments will be reported to YT.

**Reply by namealreadytak** [reply]
According to the program, you've posted that comment 136 times. Weird.

**Reply by Firecul** [reply]
If there are comments you find in one of the named 3 modes you believe are not spam then it's best to open an issue in this repo with links to the comments so ThioJoe can look at them and possibly tweak the filters to exclude them. This is also true of ones you believe *are* spam, in this case, we can change things to possibly catch them.

**Reply by Firecul** [reply]
> According to the program, you've posted that comment 136 times. Weird.

If he scrolls up you'd maybe see other messages from the same author, not that specific one 136x most likely. You see the word `offer` at the end of the last one. Seeing that, I think he was scanning for a specific person in this case.

**Reply by mrsqueakcat** [reply]
Yes, I was scanning a specific well-known troll/bot account that breaks the YT rules constantly yet never gets anything taken down no matter how often people report it.  It's one of a matching pair of YT's apparently untouchable harassment bot accounts run by an obvious troll/bot farm. 

I was hoping this program could throw enough reporting bias on it to get YT to actually do something about it for once.

**Reply by mrsqueakcat** [reply]
Unfortunately, I am finding the program interface rather clunky and not so easy to follow or use due to the lack of intuitive feel of things.

**Reply by DinhHuy2010** [reply]
wiki check: https://github.com/ThioJoe/YT-Spammer-Purge/wiki/Moderator-Mode-&-Limiting-Permissions#limitations-of-moderator-mode

**Reply by mrsqueakcat** [reply]
So this is basically useless for dealing with and reproting the more advanced troll farm/bot account operators that are running in plain sight and taking over our favorite channels?

**Reply by Firecul** [reply]
Again, if they are doing things inherently spammy, please make an issue with links to example comments so the filters can be updated to catch them.

**Reply by mrsqueakcat** [reply]
Okay. I think I am following this, but not well yet. This type of programing and software is way outside the industrial application stuff I used to work with so it is all new lingo and terminology to me. 

 Is there any way to add them to whatever filter lists it uses on my end too?   I'm assuming that people beyond me are finding new spammer channels or the same spammers just making a new channel as soon as an old one gets removed popping up and don't want to wait until some external to us master list is updated.

**Reply by mrsqueakcat** [reply]
I see some files called spamaccountslist, spamnameslist, and spamthreadslist.   Are these what it primarily goes by, and if so can new names be added to them manually to improve filtering or not?

**Reply by TechStudent10** [reply]
> I see some files called spamaccountslist, spamnameslist, and spamthreadslist. Are these what it primarily goes by, and if so can new names be added to them manually to improve filtering or not?

You can, but I wouldn't recommend it. You can open an issue in [`ThioJoe/YT-Spam-Lists`](https://github.com/ThioJoe/YT-Spam-Lists) to update the files.

**Comment by TechStudent10** [comment]
I'm confused. Has the issue been solved?

**Reply by Firecul** [reply]
It has.

**Reply by TechStudent10** [reply]
Alright

**Reply by DinhHuy2010** [reply]
what?
